credentials/fetch_paa_certs_from_dcl.py

```diff
@@ -148,36 +184,43 @@ def main(use_main_net_dcld, use_test_net_dcld, use_main_net_http, use_test_net_h
     rest_node_url = PRODUCTION_NODE_URL_REST if production else TEST_NODE_URL_REST
 
     os.makedirs(paa_trust_store_path, exist_ok=True)
+    original_dir = os.getcwd()
```

Comment:
let's update code to not rely on cwd or chdir.

Reply:
Leaving for future PR, so that we do not mix new functionality with "make code not rely on cwd"